### PR TITLE
Added image tags support in rss feeds.

### PR DESCRIPTION
**Pull request by m-pastuszek**

### CODE REVIEW

File: resources/views/feed.blade.php [hunk 1]
```diff
@@ -22,6 +22,7 @@
             </author>
             <summary type="html">
                 <![CDATA[{!! $item->summary !!}]]>
+                <img src="{{ env('APP_URL') }}/storage/{{ $item->image }}" width="800" height="450" alt="" typeof="Image" class="image-style-body-default">
```
Comment (freekmurze) [inline resources/views/feed.blade.php]: This requires users to store their images in the `storage` directory (and also requiring them to set up symlinks. I think it's better to rename`image` to `imageUrl` and let the user responsible for the entire url (so remove `storage` and the env variable).

File: src/FeedItem.php [hunk 2]
```diff
@@ -79,10 +82,17 @@ public function author(string $author)
...
+    {
+        $this->image = $image;
+
+        return $this;
+    }
 
     public function validate()
     {
-        $requiredFields = ['id', 'title', 'updated', 'summary', 'link', 'author'];
+        $requiredFields = ['id', 'title', 'updated', 'summary', 'link', 'author', 'image'];
```
Comment (SamuelNitsche) [inline src/FeedItem.php]: I don't think that the image should be required.

Comment (freekmurze) [reply src/FeedItem.php]: agreed

**Comment by sebastiandedeyne** [comment]
I'm closing this for now. Freek's comment on the hard coded storage directory is a big concern. Feel free to send over a new PR that solves the above issues.

**Pull request closed without merging**